### ISSUE TITLE
YOLO9000: Better, Faster, Stronger

### ISSUE THREAD

**Issue by NoahHalonen** [NoahHalonen/NoahHalonen-Capstone_Resources#18]
### Title
YOLO9000: Better, Faster, Stronger

### URL
http://pjreddie.com/yolo9000/

### Summary 
The paper presents YOLO9000, a state-of-the-art, real-time object detection system capable of identifying over 9,000 object categories. It introduces improvements to the YOLO detection method, enhancing speed and accuracy, and discusses a novel multi-scale training method. This allows the YOLOv2 model to operate at various sizes, balancing speed and accuracy effectively. Furthermore, YOLO9000 is trained on both the COCO detection dataset and the ImageNet classification dataset, enabling it to predict object categories that lack labeled detection data.

### Key Points 
- YOLO9000 is a significant advancement in real-time object detection, identifying over 9,000 categories.
- The paper introduces improvements to YOLO, including batch normalization, high-resolution classifiers, and a novel training method that adjusts to different image sizes.
- YOLOv2, the improved model, achieves top performance on standard detection tasks while maintaining high speed.
- A joint training method on COCO and ImageNet enables YOLO9000 to predict a wide range of object categories, even without explicit detection data for many of them.
- The approach combines detection and classification datasets using a hierarchical tree of visual concepts, facilitating the training on a broader category spectrum.

### Citation
@misc{redmon2016yolo9000,
      title={YOLO9000: Better, Faster, Stronger}, 
      author={Joseph Redmon and Ali Farhadi},
      year={2016},
      eprint={1612.08242},
      archivePrefix={arXiv},
      primaryClass={cs.CV}
}
